### PR TITLE
[zh] update /setup/platform-setup docs

### PR DESCRIPTION
**Pull request by windsonsea**
Update these docs:

```
modified:   content/zh/docs/setup/platform-setup/MicroK8s/index.md
modified:   content/zh/docs/setup/platform-setup/alicloud/index.md
modified:   content/zh/docs/setup/platform-setup/docker/index.md
modified:   content/zh/docs/setup/platform-setup/gardener/index.md
modified:   content/zh/docs/setup/platform-setup/gke/index.md
modified:   content/zh/docs/setup/platform-setup/huaweicloud/index.md
modified:   content/zh/docs/setup/platform-setup/k3d/index.md
modified:   content/zh/docs/setup/platform-setup/kops/index.md
modified:   content/zh/docs/setup/platform-setup/kubesphere/index.md
```

- [x] Docs

### CODE REVIEW

File: content/zh/docs/setup/platform-setup/oci/index.md [hunk 1]
```diff
@@ -58,7 +60,8 @@ $ oci ce cluster create-kubeconfig \
 虽然一个 OKE 集群可能暴露多个端点，但只会攻击 `kubecconfig` 文件中的那个端点。
 {{< /tip >}}
 
-`kube-endpoint` 支持的值是 `PUBLIC_ENDPOINT` 或 `PRIVATE_ENDPOINT`。您可能还需要配置 SSH 隧道通过 [Bastion 主机][bastion]访问只有私有端点的集群。
+`kube-endpoint` 支持的值是 `PUBLIC_ENDPOINT` 或 `PRIVATE_ENDPOINT`。
+您可能还需要配置 SSH 隧道通过 [Bastion 主机][bastion]访问只有私有端点的集群。
```
Comment (hanxiaop) [inline content/zh/docs/setup/platform-setup/oci/index.md:64]:
```suggestion
您可能还需要通过 [Bastion 主机][bastion]配置 SSH 隧道以访问只有私有端点的集群。
```

File: content/zh/docs/setup/platform-setup/oci/index.md [hunk 2]
```diff
@@ -58,7 +60,8 @@ $ oci ce cluster create-kubeconfig \
...
 
 将 `cluster-ocid` 替换为目标 OKE 集群的[OCID][CONCEPTS]。
```
Comment (wilsonwu) [inline content/zh/docs/setup/platform-setup/oci/index.md:66]:
```suggestion
将 `cluster-ocid` 替换为目标 OKE 集群的 [OCID][CONCEPTS]。
```

File: content/zh/docs/setup/platform-setup/k3d/index.md [hunk 3]
```diff
@@ -8,17 +8,17 @@ owner: istio/wg-environments-maintainers
...
 - 安装最新版本 [k3d](https://k3d.io/v5.4.7/#installation)。
-- 与 Kubernetes 集群 [kubectl](https://kubernetes.io/zh/docs/docs/tasks/tools/#kubectl) 进行交互。
-- (可选) [Helm](https://helm.sh/docs/intro/install/)是针对 Kubernetes 的包管理器。。
+- 与 Kubernetes 集群 [kubectl](https://kubernetes.io/zh-cn/docs/docs/tasks/tools/#kubectl) 进行交互。
+- (可选) [Helm](https://helm.sh/docs/intro/install/) 是针对 Kubernetes 的包管理器。
 
-## 安装{#installation}
+## 安装 {#installation}
 
 1. 创建集群并使用以下命令禁用 `Traefik`:
```
Comment (my-git9) [inline content/zh/docs/setup/platform-setup/k3d/index.md:23]:
```suggestion
1. 创建集群并使用以下命令禁用 `Traefik`：
```

File: content/zh/docs/setup/platform-setup/k3d/index.md [hunk 4]
```diff
@@ -71,9 +71,9 @@ k3d 使得在 docker 中创建单节点和多节点 k3s 集群变得非常容易
...
     {{< /text >}}
 
-## 为 k3d 设置仪表板用户界面{#set-up-dashboard-UI-for-k3d}
+## 为 k3d 设置仪表板用户界面 {#set-up-dashboard-UI-for-k3d}
 
-k3d 没有像 minikube 这样的内置仪表板 UI。但是您仍然可以设置 Dashboard (基于 web 的 Kubernetes UI) 来查看您的集群。
+k3d 没有像 minikube 这样的内置仪表板 UI。但是您仍然可以设置 Dashboard (基于 Web 的 Kubernetes UI) 来查看您的集群。
 按照以下说明为 k3d 设置仪表板。
 
 1. 要部署仪表板，请运行以下命令:
```
Comment (my-git9) [inline content/zh/docs/setup/platform-setup/k3d/index.md:79]:
```suggestion
1. 要部署仪表板，请运行以下命令：
```

File: content/zh/docs/setup/platform-setup/docker/index.md [hunk 5]
```diff
@@ -15,7 +15,8 @@ test: no
 1. 如果您想在 Docker Desktop 下运行 Istio，则需要安装[受支持的 Kubernetes 版本](/zh/docs/releases/supported-releases#support-status-of-istio-releases)
     ({{< supported_kubernetes_versions >}})。
 
-1. 如果您想在 Docker Desktop 内置的 Kubernetes 下运行 Istio，您可能需要在 Docker Desktop 的 *Settings...* 中的 *Resources->Advanced* 面板下增加 Docker 的内存限制。将资源设置为至少 8.0 `GB` 的内存和 4 核心 `CPUs`.
+1. 如果您想在 Docker Desktop 内置的 Kubernetes 下运行 Istio，您可能需要在 Docker Desktop 的 **Settings...** 中的
+   **Resources->Advanced** 面板下增加 Docker 的内存限制。将资源设置为至少 8.0 `GB` 的内存和 4 核心 `CPUs`。
```
Comment (Michelle951) [inline content/zh/docs/setup/platform-setup/docker/index.md:19]:
```suggestion
   **Resources->Advanced** 面板下增加 Docker 的内存限制。将资源设置为至少 8.0 `GB` 的内存和 4 核心 `CPU`。
```

Comment (windsonsea) [reply content/zh/docs/setup/platform-setup/docker/index.md:19]: 这 `CPUs` 如果写在了 UI 上，就只能跟 UI 一致，因为 Istio 产品本身没有中文版

**Review by hanxiaop — APPROVED** [review]
Looks good

**Review by my-git9 — APPROVED** [review]

**Comment by windsonsea** [comment]
Hi @wilsonwu @zirain , could we move this forward. It involves nearly ten files, and over time, it may lead to conflicts.

**Review by wilsonwu — APPROVED** [review]
lgtm

**Pull request merged**